src/lib/Tab/Dashboard/PagerLocationToDataMapper.php

```diff
@@ -84,4 +85,22 @@ private function getVersionContributor(VersionInfo $versionInfo): ?User
             return null;
         }
     }
+
+    /**
+     * @return iterable<\eZ\Publish\API\Repository\Values\Content\Language>
+     */
+    private function getAvailableTranslations(
+        VersionInfo $versionInfo
+    ): iterable {
```

Comment:
Is there any specific reason why not:
```suggestion
    ): array {
```
?

Reply:
No, I think I just adjusted some existing method, will fix.